testing/simapp/app.go

```diff
@@ -353,66 +353,78 @@ func NewSimApp(
 		&stakingKeeper, govRouter,
 	)
 
+	// Fee Module keeper
 	app.IBCFeeKeeper = ibcfeekeeper.NewKeeper(appCodec, keys[ibcfeetypes.StoreKey], app.GetSubspace(ibcfeetypes.ModuleName),
 		app.IBCKeeper.ChannelKeeper, app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper, app.AccountKeeper, app.BankKeeper,
 	)
 
+	// Create Transfer Stack
```

Comment:
Should we add a comment similar to [this](https://github.com/tharsis/evmos/blob/main/app/app.go#L488-L494)? 

I think even something simple like that makes the flow of control much clearer